control-plane/tests/test_phase28_endpoint_evidence_pack_validation.py

```
@@ -0,0 +1,324 @@
+                    "artifact_class": "collection_manifest",
+                    "artifact_id": "manifest-001",
+                    "source_artifact_id": "collector-manifest-001",
+                    "collected_at": reviewed_at,
+                    "collector_identity": "velociraptor",
+                    "tool_name": "Velociraptor",
+                    "tool_version": "0.7.2",
+                    "source_boundary": "endpoint_evidence_pack",
+                    "citation_kind": "raw_collected_output",
+                    "description": "Reviewed manifest for the bounded endpoint collection.",
+                    "content": {
+                        "requested_paths": ("/var/tmp/suspicious",),
```

Ruff (0.15.10)

[error] Probable insecure usage of temporary file or directory: "/var/tmp/suspicious"

(S108)